unstructured/file_utils/filetype.py

```diff
@@ -126,26 +129,52 @@ def __lt__(self, other):
         return self.name < other.name
 
 
+FILETYPE_TO_MIMETYPE = {
```

Comment:
The reason I went this way was because a few entries have more than one mimetype for a single filetype. Maybe it'd be better if I reversed `STR_TO_FILETYPE` and just dealt with those cases individually.

Reply:
Yeah that makes sense on the multiple MIME types point. Only real downside I see to the second mapping is that someone might update one and forget to update the other. I'm fine keeping this the way it is, just wanted to check on the reasoning.

Reply:
No I agree with you, I switched to deriving it from `STR_TO_FILETYPE` while skipping the mimetypes that would cause a duplicate `FileType` in the keys.